tick-api/src/main/java/me/aurium/tick/container/terms/JDBCTerms.java

```diff
@@ -0,0 +1,32 @@
+package me.aurium.tick.container.terms;
+
+import com.github.dockerjava.api.DockerClient;
+import com.github.dockerjava.api.command.CreateContainerResponse;
+import me.aurium.tick.container.CreationTerms;
+
+public abstract class JDBCTerms implements CreationTerms {
+
+    private final String db_name;
+    private final String db_username;
+    private final String db_password;
+
+    protected JDBCTerms(String db_name, String db_username, String db_password) {
+        this.db_name = db_name;
+        this.db_username = db_username;
+        this.db_password = db_password;
+    }
+
+    protected String getDb_name() {
```

Comment:
Method names should not usually include underscores. Neither should variable names